IsoCountries.swift

```diff
@@ -25,8 +25,8 @@ class IsoCountries {
     class func flag(countryCode: String) -> String {
         var string = ""
         var country = countryCode.uppercased()
-        for uS in country.unicodeScalars {
-            string.unicodeScalars.append(UnicodeScalar(127397 + uS.value)!)
+        for scalar in country.unicodeScalars {
+            string.unicodeScalars.append(UnicodeScalar(127397 + scalar.value)!)
```

Comment:
Something along the lines of this:
```
guard let flag = UnicodeScalar(127397 + scalar.value) else { break }
string.unicodeScalars.append(flag)
```
(untested in and editor - doing it live in Github)